sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -554,14 +561,19 @@ case class FromUnixTime(sec: Expression, format: Expression)
           boolean ${ev.isNull} = true;
           ${ctx.javaType(dataType)} ${ev.value} = ${ctx.defaultValue(dataType)};""")
       } else {
+        val sdfTerm = ctx.freshName("formatter")
```

Comment:
This is trivial but why use a different variable name here from the above one? (which is called "formatter")